workflow-templates/prerelease.yml

```diff
@@ -25,11 +25,33 @@ jobs:
     needs: prerelease
     runs-on: ubuntu-20.04
     if: needs.prerelease.outputs.key
-    environment:
```

Comment:
`environment` nya tetep ada @bayyuaji , ini beda sama `env`
bisa liat di https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idenvironment